packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -361,18 +380,23 @@ async function launchDesktopApp(app: OfficeApp, manifest: ManifestInfo, document
     throw new ExpectedError(`Sideload to the ${getOfficeAppName(app)} app is not supported.`);
   }
 
-  // for Outlook, open Outlook.exe; for other Office apps, open the document
+  // for Outlook, Word, Excel, PowerPoint open {Host}.exe; for other Office apps, open the document
+  let path: string;
   if (app == OfficeApp.Outlook) {
```

Comment:
This should be combined with the if manifestType = json

Reply:
Yes. Will change.